components/_global-navigation.scss

```diff
@@ -300,13 +308,18 @@ $notification-counter-size: 18px;
 			flex: 1 1 0;
 		}
 
-		&__search-input {
-			display: block;
-		}
-
 		&__search-input-wrapper {
 			border-bottom-width: 1px;
 		}
+
+		&__search-label-icon:hover {
+			color: $color-dark-blue-gray;
```

Comment:
Can we do it in a way that won't need change in multiple places when we decide to change this default color? E.g. use a variable for all navigation default colors.